projects/python-examples/algokit_utils_py_examples/transactions/transaction_types.py

```diff
@@ -0,0 +1,417 @@
+    - receiver: The address of the account to send the asset to
+    """
+    asset_transfer_txn = algorand_client.create_transaction.asset_transfer(
+        AssetTransferParams(
+            sender=account1.address,
+            asset_id=1234,
+            receiver=account2.address,
+            amount=1,
+        )
+    )
+
+    # example: ASSET_TRANSFER_TRANSACTION
+
+    # example: ASSET_OPT_IN_TRANSACTION
+
+    """
+    Create an unsigned asset opt in transaction for account1 opting in to asset with asset id 1234
+
+    Parameters for an asset opt in transaction.
+    - sender: The address of the account that will opt in to the asset
+    - asset_id: ID of the asset
+    """
+    asset_opt_in_txn = algorand_client.create_transaction.asset_opt_in(
+        AssetOptInParams(
+            sender=account1.address,
+            asset_id=1234,
+        )
+    )
+
+    # example: ASSET_OPT_IN_TRANSACTION
+
+    # example: ASSET_OPT_OUT_TRANSACTION
+
+    """
+    Create an unsigned asset opt out transaction for account1 opting out of asset with asset id 1234
+
+    Parameters for an asset opt out transaction.
+    - sender: The address of the account that will opt out of the asset
+    - asset_id: ID of the asset
+    - creator: The creator address of the asset
+    """
+    asset_opt_out_txn = algorand_client.create_transaction.asset_opt_out(
+        AssetOptOutParams(
+            sender=account1.address,
+            asset_id=1234,
+            creator=account2.address,
+        )
+    )
+
+    # example: ASSET_OPT_OUT_TRANSACTION
+
+    # example: ASSET_CREATE_TRANSACTION
+
+    """
+    Create an unsigned asset create transaction creating a fungible ASA with 10 million units
+
+    Parameters for creating a new asset.
+    - sender: The address of the account that will send the transaction
+    - total: The total amount of the smallest divisible unit to create
+    - decimals: The amount of decimal places the asset should have, defaults to None
+    - default_frozen: Whether the asset is frozen by default in the creator address, defaults to None
+    - manager: The address that can change the manager, reserve, clawback, and freeze addresses, defaults to None
+    - reserve: The address that holds the uncirculated supply, defaults to None
+    - freeze: The address that can freeze the asset in any account, defaults to None
+    - clawback: The address that can clawback the asset from any account, defaults to None
+    - unit_name: The short ticker name for the asset, defaults to None
+    - asset_name: The full name of the asset, defaults to None
+    """
+    asset_create_txn = algorand_client.create_transaction.asset_create(
+        AssetCreateParams(
+            sender=account1.address,
+            total=10_000_000,
+            decimals=6,
+            default_frozen=False,  # optional
+            manager=account1.address,  # optional. Can be permanently disabled by setting to None
```

Comment:
in my case I was passing in `Account` rather than the address... not sure which is better in practice, maybe we revisit and settle on one method over the other once it's in the docs. I believe utils caches the signers either way?

Reply:
for Python they have to be `str` type. So I have to pass in addresses